app/helpers/application_helper.rb

```diff
@@ -1,2 +1,25 @@
 module ApplicationHelper
+  module ActionView
+    module Helpers
+      class FormBuilder
+        def date_select(method, options = {}, html_options = {})
+          existing_date = @object.send(method)
+          formatted_date = existing_date.to_date.strftime("%F") if existing_date.present?
+          @template.content_tag(:div, :class => "input-group") do
+            text_field(method, :value => formatted_date, :class => "form-control datepicker", :"data-date-format" => "YYYY-MM-DD") +
+            @template.content_tag(:span, @template.content_tag(:span, "", :class => "glyphicon glyphicon-calendar") ,:class => "input-group-addon")
```

Comment:
Line is too long. [147/80]<br>Use the new Ruby 1.9 hash syntax.<br>Use 2 (not 0) spaces for indenting an expression spanning multiple lines.<br>Space missing after comma.<br>Space found before comma.